src/SixLabors.Fonts/Tables/General/Glyphs/GlyphVector.cs

```diff
@@ -110,7 +129,7 @@ public static GlyphVector Append(GlyphVector first, GlyphVector second)
             }
 
             var bounds = new Bounds(first.Bounds.Min, first.Bounds.Max);
-            return new GlyphVector(controlPoints, onCurves, endPoints, bounds);
+            return new GlyphVector(controlPoints, onCurves, endPoints, bounds, first.Instructions);
```

Comment:
I wonder if we should be appending the instructions here?

Reply:
not that I've been able find anything to back up my theory, but my guess would be that we should be hinting each transformed composite glyph part independently and combing them post hinting/as we render.

Just munging the instruction sets together I can't see working as the instructions would have be written against the set of control points in the part not the new composite set.

Reply:
I wonder how that would work for word though, we’d have to refactor the way we store the glyphs there. We can have a peek through the FreeType source to see if we can figure out what they do. (Hard though, the code seems scattered)

On that note I think our fallback top side bearing calculation might be off. I’ll dig out the example I found when I’m at my computer later.

Reply:
![image](https://user-images.githubusercontent.com/385879/135453546-3e771125-b6e5-433a-88ae-a00649e115b6.png)


@tocsoft Looks like you're on to something.